### PR TITLE
Fix missing icons and wrong styling for download menu

### PR DESCRIPTION
**Pull request by csidirop**
This PR fixes some missing icons and corrects the styling for the `imagedownloadtool` and `fulltextdownloadtool`:

grafik
The `fulltextdownloadtool` missed an icon if there was no fulltext. 
 
grafik
The `imagedownloadtool` had no icon at all. So I reused one which belongs to the pdf page download button. Furthermore it fixed the opacity. 

And in the doublepageview:
all

As you can see, for some reason the imagedownloadtool had a second `::before` element and was wrapped in a `span`. I was not able to find the reason for the existence of the second element, so maybe it was mistake?
I have tried to work around the problem, at least for now, by hiding it and moving the text. Only drawback is that in the doublepageview the download for the second page image is now missing all `::before` elements.

What do you think? 

Also I left both tools active. I thought that since DFG-Viewer is a reference implementation, it should also show what it can do by default. But thats only my opinion and can be easily undone.

### CODE REVIEW

**Comment by csidirop** [comment]
In addition to this changes, this PR https://github.com/slub/slub_digitalcollections/pull/65 fixes the page pdf icon

File: Configuration/TypoScript/Plugins/kitodo.typoscript [hunk 1]
```diff
@@ -188,7 +188,8 @@ plugin.tx_dlf_imagemanipulationtool {
 plugin.tx_dlf_pdfdownloadtool < plugin.tx_dlf_toolbox
 plugin.tx_dlf_pdfdownloadtool {
     settings {
-        tools = pdfdownloadtool
+        fileGrpsImageDownload = DEFAULT
+        tools = tx_dlf_pdfdownloadtool,imagedownloadtool,fulltextdownloadtool
```
Comment (beatrycze-volk) [inline Configuration/TypoScript/Plugins/kitodo.typoscript:192]: Could you use unified naming for tools - `tx_dlf_xyztool` or `xyztool`?

**Review by beatrycze-volk — CHANGES_REQUESTED** [review]

File: Configuration/TypoScript/Plugins/kitodo.typoscript (hunk 1 above)
Comment (beatrycze-volk) [reply Configuration/TypoScript/Plugins/kitodo.typoscript:192]: I prefer the shorter version but feel free to choose whatever fits you better :)

**Review by beatrycze-volk — APPROVED** [review]
Thanks for the changes!

**Pull request merged**